### PR TITLE
Use Google+ API to get user profile image

### PR DESCRIPTION
**Pull request by Happy-Neko**
Fix for #431: `PsGoogle` uses Google+ API to get user profile image.

### CODE REVIEW

**Comment by Happy-Neko** [comment]
@davvd Please proceed with code review.

**Comment by davvd** [comment]
@happy-neko Thanks, I will ask someone to do a code review here, thanks for your pull request

**Comment by davvd** [comment]
@dmzaytsev it's yours, please review

File: src/test/java/org/takes/facets/auth/social/PsGoogleTest.java [hunk 1]
```diff
@@ -104,7 +104,7 @@ public Response act(final Request req) throws IOException {
                         MatcherAssert.assertThat(
                             new RqPrint(req).printHead(),
                             Matchers.containsString(
-                                "GET /oauth2/v1/userinfo"
+                                "GET /plus/v1/people/me"
```
Comment (dmzaytsev) [inline src/test/java/org/takes/facets/auth/social/PsGoogleTest.java]: @Happy-Neko we need check parameters as well
if parameter `image/url` is needed

File: src/main/java/org/takes/facets/auth/social/PsGoogle.java [hunk 2]
```diff
@@ -143,9 +143,9 @@ public Response exit(final Response response,
      */
     private Identity fetch(final String token) throws IOException {
         // @checkstyle LineLength (1 line)
-        final String uri = new Href(this.gapi).path("oauth2").path("v1")
-            .path("userinfo")
-            .with("alt", "json")
+        final String uri = new Href(this.gapi).path("plus").path("v1")
+            .path("people")
+            .path("me")
             .with("access_token", token)
```
Comment (dmzaytsev) [inline src/main/java/org/takes/facets/auth/social/PsGoogle.java]: @Happy-Neko on the screenshot in the original issue you can see a selector `image/url` in parameters
will the api call work without this parameter?
did you try this api call on real server?

**Comment by dmzaytsev** [comment]
@Happy-Neko please see two comments

**Comment by Happy-Neko** [comment]
@dmzaytsev `fileds` param is optional.
Tested [here](https://developers.google.com/oauthplayground).

**Comment by dmzaytsev** [comment]
@Happy-Neko okay, thanks!

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by davvd** [comment]
@ypshenychka please, help us to evaluate quality here, as per [par.24](http://at.teamed.io/policy.html#24)

**Comment by davvd** [comment]
@rultor deploy

**Comment by ypshenychka** [comment]
@dmzaytsev 
According to our [QA Rules](http://www.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by dmzaytsev** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@dmzaytsev thank you!

**Comment by ypshenychka** [comment]
@davvd Quality is acceptable.

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@ypshenychka thanks a lot, next time everybody should try to make it better

**Comment by davvd** [comment]
@dmzaytsev added 10 mins to @ypshenychka, for QA review, payment code is `72998569`

Thanks for your contribution, **21 mins** was added to your account, payment ID is `72998576`, 17 hours and 34 mins spent total

you're getting extra minutes here (c=6)

added +21 to your rating, now it is equal to [+1164](http://www.netbout.com/b/35166?open=rating)

**Pull request merged**